lib/dpul_collections/indexing_pipeline/figgy/transformation_consumer.ex

```diff
@@ -49,6 +49,24 @@ defmodule DpulCollections.IndexingPipeline.Figgy.TransformationConsumer do
   @impl Broadway
   @spec handle_message(any(), any(), %{required(:cache_version) => integer()}) ::
           Broadway.Message.t()
+  def handle_message(
```

Comment:
We think we can remove this, the function is the same it's just the document gets created differently